scripts/src/java/org/oppia/android/scripts/lint/LintModelCreator.kt

```diff
@@ -245,13 +245,13 @@ class LintModelCreator(
     val content =
       """
         <lint-module
-            dir="${escapeXmlAttribute(relativeProjectPath.toString())}"
+            dir="${escapeXmlAttribute(modulePath.toString())}"
             name="${escapeXmlAttribute(moduleConfig.name)}"
             type="${moduleType.name}"
-            maven="__non_maven__"
             buildFolder="${escapeXmlAttribute(buildDir.toFile().absolutePath)}"
```

Comment:
Should we ensure build directory exists before referencing it here? 

Double check this but I think if we just do `buildDir.createDirectories().toFile()`, it will automatically create (if not present) or use the same one if present.